### PR TITLE
Userinterface

### PR DESCRIPTION
**Pull request by PhuocBao**

### CODE REVIEW

File: app/src/main/res/menu/menu.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        android:id="@+id/menu_classical"
+        android:icon="@drawable/icon_music"
+        android:title="@string/classical"/>
+
+    <item
+        android:id="@+id/menu_country"
+        android:icon="@drawable/icon_music"
+        android:title="@string/country"/>
+    >>>>>>> 58e8606... fixed
+</menu>
```
Comment (le-quang-dao) [inline app/src/main/res/menu/menu.xml]: WARNING: [ExtraText] Unexpected text found in layout file: ">>>>>>> 58e8606... fixed"
 Layout resource files should only contain elements and attributes. Any XML text content found \
in the file is likely accidental (and potentially dangerous if the text resembles XML and the \
developer believes the text to be functional)

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="open">Open</string>
+    <string name="close">Close</string>
+    <string name="allmusic">All music genres</string>
+    <string name="allaudio">All audio genres</string>
+    <string name="rock">Alternative Rock</string>
+    <string name="ambient">Ambient</string>
+    <string name="classical">Classical</string>
+    <string name="country">Country</string>
+    <string name="search">Search</string>
```
Comment (le-quang-dao) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.search` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/java/com/framgia/music/ViewPagerAdapter.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            case HOME_ITEM:
+                return mContext.getResources().getString(R.string.home_tab);
+            case MYMUSIC_ITEM:
+                return mContext.getResources().getString(R.string.mymusic_tab);
+            default:
+                return null;
+        }
+    }
+
+    public @interface TabItem {
```
Comment (ThuanPx) [inline app/src/main/java/com/framgia/music/ViewPagerAdapter.java]: thiếu rồi e https://developer.android.com/reference/android/support/annotation/IntDef.html

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -1,21 +1,34 @@
...
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
+        android:layout_height="match_parent"
+        android:layout_gravity="start"
+        app:menu="@menu/menu"
         />
-
-</android.support.constraint.ConstraintLayout>
+</android.support.v4.widget.DrawerLayout>
```
Comment (ThuanPx) [inline app/src/main/res/layout/activity_main.xml]: enter

File: app/src/main/res/menu/menu.xml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    <item
+        android:id="@+id/menu_classical"
+        android:icon="@drawable/icon_music"
+        android:title="@string/classical"/>
+
+    <item
+        android:id="@+id/menu_country"
+        android:icon="@drawable/icon_music"
+        android:title="@string/country"/>
+</menu>
```
Comment (ThuanPx) [inline app/src/main/res/menu/menu.xml]: enter

File: app/src/main/res/values/dimens.xml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dp_280">280dp</dimen>
+    <dimen name="dp_170">170dp</dimen>
+</resources>
```
Comment (ThuanPx) [inline app/src/main/res/values/dimens.xml]: enter

File: app/src/main/java/com/framgia/music/MainActivity.java [hunk 7]
```diff
@@ -1,13 +1,63 @@
...
+    public boolean onOptionsItemSelected(MenuItem item) {
+        return mActionBarDrawerToggle.onOptionsItemSelected(item);
+    }
+
+    private void initView() {
+        mDrawerLayout = findViewById(R.id.drawerlayout_main);
+    }
+
+    private void initToolbar() {
+        Toolbar mToolbar = findViewById(R.id.toolbar_main);
```
Comment (ThuanPx) [inline app/src/main/java/com/framgia/music/MainActivity.java]: Toolbar không phải là 1 view a e

File: app/src/main/java/com/framgia/music/MainActivity.java [hunk 8]
```diff
@@ -1,13 +1,63 @@
...
+        setSupportActionBar(mToolbar);
+        setTitle(R.string.title);
+    }
+
+    private void initViewPager() {
+        TabLayout mTabLayout = findViewById(R.id.tablayout_main);
```
Comment (ThuanPx) [inline app/src/main/java/com/framgia/music/MainActivity.java]: Tablayout cũng là 1 view mà

**Pull request closed without merging**